### PR TITLE
feat: convert cases-ms to typescript

### PR DESCRIPTION
**Pull request by petter-a**
6th API converted to typescript

### CODE REVIEW

File: services/case-ms/src/lambdas/updatePdf.ts [hunk 1]
```diff
@@ -1,13 +1,25 @@
+/* eslint-disable no-console */
 import to from 'await-to-js';
 
-import config from '../../../config';
+import config from '../libs/config';
```
Comment (dannil76) [inline services/case-ms/src/lambdas/updatePdf.ts:4]: Not sure if this is a good way to salve it using symlinks?

Please let's have a talk :)

**Review by dannil76 — CHANGES_REQUESTED** [review]

File: services/case-ms/src/lambdas/updatePdf.ts (hunk 1 above)
Comment (petter-a) [reply services/case-ms/src/lambdas/updatePdf.ts:4]: typescript expects source code to be located in a single folder tree, usually with the root named src. Crosslinking outside this path comes with a lot of caveats and hence symlinking is preferred.

File: services/case-ms/src/lambdas/updatePdf.ts [hunk 2]
```diff
@@ -47,7 +59,7 @@ export async function main(event, context) {
   return true;
 }
 
-function updateCasePdfAttributes(caseKeys, pdf) {
+function updateCasePdfAttributes(caseKeys: CaseKeys, pdf: string) {
```
Comment (dannil76) [inline services/case-ms/src/lambdas/updatePdf.ts:62]: Option:
If the args have a clear type hinting, could the args itself be trimmed to just keys? :)

**Review by dannil76 — CHANGES_REQUESTED** [review]

File: services/case-ms/src/lambdas/updatePdf.ts (hunk 2 above)
Comment (petter-a) [reply services/case-ms/src/lambdas/updatePdf.ts]: Done

**Review by dannil76 — APPROVED** [review]
:)

**Pull request merged**